### ISSUE TITLE
Bug getting childs of virtual nodes 

### ISSUE THREAD

**Issue by Unsigno** [ErikSchierboom/knockout-pre-rendered#48]
Hi, first of all, thanks for the library, looks very useful. But I noticed some problems using it on virtual fields. 

Getting these two examples, we should expect a equivalent result . 
*I edited the code here to keep it simple, so can have some typographical mistake

The JS is not relevant here, and i used the same for the two tests .

```
<script>
    function ItemModel(id){
        var self = this;
        self.id = ko.observable(id);
    }
    function ViewModel(){
        var self = this;
        self.items = ko.observableArray();
        self.createItem = function(){
            return new ItemModel();
        }
    }
    var viewModel = new ViewModel();
    ko.applyBindings(viewModel);
</script>
```

Case 1 ( "external" template )
```
<ul>
    <li>Header</li>
    <!-- ko foreachInit: { data: items, name: "itemTemplate", createElement: createItem } -->
        <li data-bind='init, text: id'>Item1</li>
        <li data-bind='init, text: id'>Item2</li>
        <li data-bind='init, text: id'>Item3</li>
    <!-- /ko -->
</ul>
<script type="text/ko-template" id="itemTemplate">
    <li data-bind='init, text: id'></li>
</script>
```

Case 2 ( "internal" template )
```
<ul>
    <li>Header</li>
    <!-- ko foreachInit: { data: items, createElement: createItem } -->
        <li data-template data-bind='init, text: id'></li>
        <li data-init data-bind='init, text: id'>Item1</li>
        <li data-init data-bind='init, text: id'>Item2</li>
        <li data-init data-bind='init, text: id'>Item3</li>
    <!-- /ko -->
</ul>
```

The "Case 2" is working well, getting the elements marked with "data-init". But in the "Case 1" is getting the "li" element used as header, as a "item" . Looking documentations and library code this seems a bug, since we should have the same result, receiving the childrens of the virtual node

**Comment by ErikSchierboom** [comment]
Thanks for the bug report. I'll have a look at at, although it can take a little while due to a holiday.

**Comment by ErikSchierboom** [comment]
@Unsigno I've just published a new version, 0.9.1-beta, that should fix this problem. Would you be willing to test it?

**Comment by Unsigno** [comment]
Hi @ErikSchierboom ,

I have tried the 2 cases and they are working as expected, so it seems to be fixed .

Thank you very much for sharing your time .

**Comment by ErikSchierboom** [comment]
Thanks for testing! I've just published the final 0.9.1 version. 🎉
